t/ui/10-tests_overview.t

```diff
@@ -382,6 +382,12 @@ subtest "job dependencies displayed on 'Test result overview' page" => sub {
       'parent job was highlighted correctly';
 };
 
+subtest "multiple groups" => sub {
+    $driver->get('/tests/overview?groupid=1001&groupid=1002');
```

Comment:
I don't think there's much JavaScript involved here so it would better to add this test to `t/10-tests_overview.t` and just check whether the DOM contains these elements.